libraries/chain/webassembly/privileged.cpp

```diff
@@ -152,28 +153,30 @@ namespace eosio { namespace chain { namespace webassembly {
    }
 
    void interface::set_finalizers(span<const char> packed_finalizer_set) {
```

Comment:
shouldn't we use `std::span` instead of just `span`? Same with `vector` below.

Reply:
This not `std::span` but `using span = eosio::vm::span<T, Extent>;`

Reply:
Oh, missed that, thanks. Still `vector` should be std::vector, right?

Reply:
yes, changed to `std::vector`